app/javascript/packs/hideBookmarkButtons.js

```diff
@@ -0,0 +1,10 @@
+/* global userData */
+const currentUser = userData();
+const currentUserId = currentUser && currentUser.id;
```

Comment:
Same non-blocking suggestion as above (optional chaining)

Reply:
Again, leaving as-is.